### PR TITLE
Bug 1273350 - make click open the secondary nav instead of navigating

### PR DESCRIPTION
**Pull request by jwforres**
Makes the hover menu accessible, fixes behavior on ipad.

### CODE REVIEW

**Comment by jwforres** [comment]
@spadgett review

@sg00dwin fyi

File: assets/app/views/_sidebar-main-nav-item.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <li ng-class="{ active: item.isSelected() }">
-  <a ng-href="{{item.href()}}" ng-click="item.click($event);" ng-attr-data-toggle="{{(item.tabs && item.tabs.length > 1) ? 'dropdown-hover' : undefined}}" ng-class="{'dropdown-toggle':(item.tabs && item.tabs.length > 1) }">
+  <a ng-href="{{item.href()}}" ng-click="item.click($event);" ng-attr-data-toggle="{{(item.tabs && item.tabs.length > 1) ? 'dropdown' : undefined}}" ng-attr-data-hover="{{(item.tabs && item.tabs.length > 1) ? 'dropdown' : undefined}}" ng-class="{'dropdown-toggle':(item.tabs && item.tabs.length > 1) }">
```
Comment (spadgett) [inline assets/app/views/_sidebar-main-nav-item.html]: Does

```
{{(item.tabs.length > 1) ? 'dropdown' : undefined}}
```

work without the `items.tab` test?

**Comment by spadgett** [comment]
One nit, otherwise LGTM.

**Comment by jwforres** [comment]
[merge]

**Pull request merged**